lib/_emerge/main.py

```diff
@@ -1033,6 +1043,42 @@ def parse_opts(tmpcmdline, silent=False):
 
         myoptions.jobs = jobs
 
+    if myoptions.jobs_tmpdir_blocks_threshold == "True":
```

Comment:
Just coming back to this: all of this really makes me think we need to revisit the option handling :(

But it's of course fine for now

Reply:
`True` here indicates a missing argument, and this can mean different things for different options. For example, for --jobs a missing argument means unlimited jobs, and so it's handled like this instead:
```python
        if myoptions.jobs == "True":
            jobs = True
```